includes/scripts/pruneflips.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+/usr/bin/python3 manage.py pruneflips && sleep $1
```

Comment:
How about using a loop here that each sleep the `CONTAINER_PRUNE_INTERVAL` instead of having supervisord restart the process "over and over" again.

Reply:
You're right. I shouldn't be using supervisord for something it wasn't built and a loop doesn't produce unnecessary log entries